### ISSUE TITLE
Commands with modifier keys don't work with Focus Blue braille displays

### ISSUE THREAD

**Issue by xtstoll** [nvaccess/nvda#7706]
### Steps to reproduce:
> 1. Install [driver](https://www.freedomscientific.com/Downloads/Focus/FocusBlueBrailleDisplayDriver) for Focus Blue braille displays
> 2. Start NVDA and configure the braille settings for the Freedom Scientific Focus (see my configs attached in screenshot)
> 3. Open Firefox or Chrome
> 4. Press the equivalent of ALT+D on the Braille display  (press SPACE + DOTS 6-8, then type the letter D in Braille)
> 5. Press the equivalent of INSERT+T on the Braille display (press SPACE + DOTS 2-8, then type the letter T in Braille)
> 6. Press the equivalent of CTRL+N on the Braille display to open a new window (press SPACE + DOTS 3-8, then type the letter N in Braille)

### Expected behavior:
> ALT+D command on the Braille display should go to the address bar
> INSERT+T command on the Braille display should read the title of the current window
> CTRL+N command on the Braille display should open a new window

### Actual behavior:
> ALT+D command on the Braille display just types the letter D
> INSERT+T command on the Braille display just types the letter T
> CTRL+N command on the Braille display just types the letter N

### System configuration:
NVDA version: 2017.3 (Brazilian Portuguese)
NVDA Installed or portable: installed
Windows version: 10
Braille display: Focus 80 Blue and Focus 40 Blue
Focus Blue driver: version 1.0.2030

Does the issue still occur after restarting your PC? Yes
Have you tried any other versions of NVDA? Yes, 2018.rc1 (same result)

![nvda_2017-10-27_14-03-27](https://user-images.githubusercontent.com/24814666/32114438-4fb0e544-bb22-11e7-8cb5-717caf4a081b.png)

**Comment by bramd** [comment]
I don't know if we have support for such prefix keys to "hold" a modifier such as ctrl. We could implement this as part of #2945.

**Comment by Adriani90** [comment]
@xtstoll the situation might have changed since the additional Freedom Scientific driver is not needed anymore. The braille display should work natively with NVDA now.
Could you please give an update? Are you still having this issue?

**Comment by xtstoll** [comment]
@Adriani90 - Sorry for the delay. I don't have easy access to the Focus 40 Blue anymore, so I needed to go to the public braille library here in São Paulo to retest this issue. 

I retested with NVDA 2019.3.1 with Windows 10. I did not use the Freedom Scientific driver, as it was stated that it's not necessary anymore with the latest NVDA. 

Unfortunately, the 2-step keystrokes don't worry. For example, to reproduce CTRL + N on the braille display to open a new window, I pressed SPACE + DOT 3 + DOT 8, then type the letter N in braille (DOTS 1-3-4-5). Nothing happened.

Simple keystrokes, like Tab, do work. For example, I pressed SPACE + DOT 4 + DOT 5 and it worked like the key Tab perfectly.

It seems the problem is still with support for the modifier keys. I am referring to the list on pages 24 and 25 of the Freedom Scientific manual, link below.

https://support.freedomscientific.com/Content/Documents/Manuals/Focus/Focus40Blue/Focus40-Blue-Users-Guide.pdf
